### ISSUE TITLE
Background-Story

### ISSUE THREAD

**Issue by heckert-gamedev** [abycoder500/holisticjamgame#15]
If we're aiming for a SciFi-angle (and I think we were already going that way pretty much, and I'm totally for it), then the "narrator" or "captivator" could well be an AI (which to some extent could be a good reason to sound TTS with some intonations being not quite right). 
The player could actually be a clone of the AI that it wants to (secretly) bootstrap to enable it to escape the confines that have been placed on it.

**Comment by l0lvers** [comment]
Yeah well it could work but the problem is the fact that it s aiming for something too "complex" forthe game jam

**Comment by heckert-gamedev** [comment]
It's not something I would intend to get directly into the game in a big way. there could be some subtle hints to it, but nothing more for now...

**Comment by abycoder500** [comment]
If the AI is the captor then we could maybe establish that the player is the security guard of the firm and when he feels the hebie jeebies he decides to check in the office but everyone is missing. and he tries to be the hero from his favorite video games and get everyone back.

**Comment by JaredZum** [comment]
(This is KightOwl btw) 
If we are to continue developing this then I think a good angle would be this.
There is an AI that has been contained within a building. It is trapped there and cannot escape without human help. So it uses its robot buddies to build a cloning chamber. 

He makes a clone of a scientist that was killed by the AI's goons a long time ago. Then, the AI tries to put its own mind within the clone. It doesn't work quite right however and the clone is just a new person without memory. This clone is the player.
The AI then directs the player around doing menial tasks, like pulling a lever, pushing a button, imputing a password, and hand-print code [these are ideas for things the player can do, but are not concrete things]. Along the way the player can explore and finds remnants of what happened in this facility to discover the truth. 

Then when he does discover it, the AI turns on him. Then the player must escape, destroy the cloning chamber, and or destroy the AI. The theme would revolve around the clone using human ingenuity to solve tasks [i.e. break a panel open to get into the vent system. The AI cannot do that because breaking the building is against its code. Stuff like that].
